webui/react/src/components/CreateUserModal.tsx

```diff
@@ -200,7 +199,7 @@ const CreateUserModalComponent: React.FC<Props> = ({
             label={USER_NAME_LABEL}
             name={USER_NAME_NAME}
             required
-            validateTrigger={['onSubmit']}>
+            validateTrigger={['onSubmit', 'onChange']}>
```

Comment:
`validateTrigger` can be removed. we can still get the same behavior i think

Reply:
OK, yeah -- looks like the [default value](https://ant.design/components/form#form) is `['onChange']` and form submit _probably_ shouldn't change the validation state of the form anyway.